SQL/database_changelog.txt

```diff
@@ -1,3 +1,23 @@
+28 June 2017, by oranges
+Added schema_revision to store the current db revision, why start at 3.0?
+
+because:
+15:09 <+MrStonedOne> 1.0 was erro, 2.0 was when i removed erro_, 3.0 was when jordie made all the strings that hold numbers numbers
+
+CREATE TABLE `schema_revision` (
+`major` TINYINT(3) UNSIGNED NOT NULL ,
```

Comment:
By the time this is a problem 64261 schema changes in the future we'll have invented time travel and can come back to get stuck in a causal time loop changing this.